pydash/pydash_web/controller/register_dashboard.py

```diff
@@ -0,0 +1,32 @@
+from flask import jsonify
+from flask_login import current_user
+from flask_restplus.reqparse import RequestParser
+
+import pydash_app.dashboard
+import pydash_logger
+
+
+logger = pydash_logger.Logger(__name__)
+
+
+def register_dashboard():
+    args = _parse_arguments()
+
+    name = args['name']
+    url = str(args['url'])
+    token = args['token']
+
```

Comment:
It be a good idea to check whether all of the arguments were present in the POST-request. (see `controller.login.py` for an example)

Reply:
Also, why only convert the url to a string? All argument should be strings already.